src/circuits/joinsplit.tcc

```diff
@@ -25,35 +25,43 @@ template<
 class joinsplit_gadget : libsnark::gadget<FieldT>
 {
 private:
-    // Multipacking gadgets for the inputs (nullifierS, commitmentS, val_pub_in,
-    // val_pub_out) (the root is a field element) `NumInputs + NumOutputs`
-    // because we pack the nullifiers (Inputs of JS = NumInputs), the
-    // commitments (Output of JS = NumOutputs) AND the v_pub_out taken out of
-    // the mix (+1) AND the public value v_pub_in that is put into the mix (+1)
-    // AND the signature hash h_sig (+1) AND the message authentication tags
-    // h_iS (+NumInputs)
+    // Number of residual bits from packing of hash digests into smaller
+    // field elements to which are added the public value of size 64 bits
+    const size_t length_bit_residual =
+        2 * 64 + (HashT::get_digest_len() - FieldT::capacity()) *
```

Comment:
Define 64 (public values bit length) as a constant and use it here

Reply:
Changed (and other 64s in the file) to `ZETH_V_SIZE` from `zeth.h`